.github/workflows/oclif-release.yml

```diff
@@ -0,0 +1,64 @@
+name: Oclif Release
+on:
+  workflow_dispatch:
+    inputs:
+      home-brew-branch:
+        description: 'Run workflow in home brew branch'
+        default: main
+      tag-name:
+        description: 'Tag released recently'
+        required: true
+      formula:
+          description: 'Formula to release in homebrew'
+          default: twilio
+      pre-release:
+        description: 'Homebrew release type'
+        default: 'false'
+jobs:
+  oclif-release:
+    name: Publish for ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    outputs:
+      sha256: ${{ steps.sha256.outputs.sha256 }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: macos-latest
+            asset_name: ${{ github.event.inputs.formula }}-v${{ github.event.inputs.tag-name }}
+            command_name: npx oclif-dev pack
+            publish: homebrew
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm cache clear & npm ci
+      - run: |
+          if [ "$RUNNER_OS" == "macOS" ]; then
+            brew install makensis
+          fi
+      - run: ${{ matrix.command_name }}
+      - name: Configure AWS credentials from Test account
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Publish assests to s3 and Caliculate sha of oclif dev pack
+        id: sha256
+        if: ${{matrix.publish == 'homebrew'}}
+        run: |
+           npx oclif-dev publish
+           brew install coreutils
+           echo "::set-output name=sha256::$(sha256sum dist/${{ matrix.asset_name }}/${{ matrix.asset_name }}.tar.gz | awk '{print $1}')"
+  home-brew-release:
+    name: Trigger homw brew release workflow
+    runs-on: ubuntu-latest
+    needs: [oclif-release]
+    steps:
+      - name: Invoke HomeBrew workflow
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: Homebrew Release
+          token: ${{ secrets.REPO_ACCESS_TOKEN }}
+          repo: LakshmiRavali/homebrew-brew
```

Comment:
Is this the branch which has a PR raised? Once the PR's merged will it be updated to main?

Reply:
Thanks for pointing, I need to change it to `twilio/homebrew-brew` (Missed as part of copying from my forked repo).